marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/graph/graphSearch.sjs

```diff
@@ -59,9 +59,29 @@ const withRelatedQuery = {
 const resultsTest4 = searchNodes(withRelatedQuery);
 
 assertions.concat([
-  test.assertEqual(4, resultsTest4.total),
+  test.assertEqual(5, resultsTest4.total),
```

Comment:
Why the total is 5 now? I don't see any new Customer, Product or BabyRegistry being added to the test data.

Reply:
thanks Aniela, I was adding for error a new Customer in Office file, I deleted it